src/main/java/io/nats/client/impl/NatsConnection.java

```diff
@@ -621,8 +645,7 @@ void handleCommunicationIssue(Exception io) {
         // Spawn a thread so we don't have timing issues with
         // waiting on read/write threads
         executor.submit(() -> {
-            try {
-                this.closeSocket(true);
+            try {this.closeSocket(true);

```

Comment:
Doesn't matter that much, but was this indent change intended?

Reply:
I must have left it after I considered a change here. Will fix.